comp/core/workloadmeta/collectors/internal/containerd/event_builder_test.go

```diff
@@ -53,6 +53,7 @@ func TestBuildCollectorEvent(t *testing.T) {
 	}
 
 	exitCode := uint32(137)
+	expectedExitCode := int32(exitCode)
 	exitTime := time.Now()
 	fakeExitInfo := &exitInfo{exitCode: &exitCode, exitTS: exitTime}
```

Comment:
Is this the sign that the type of `exitCode` inside the `exitInfo` struct should also be updated ?

```suggestion
	exitCode := int32(137)
	exitTime := time.Now()
	fakeExitInfo := &exitInfo{exitCode: &exitCode, exitTS: exitTime}
```

Reply:
https://github.com/DataDog/datadog-agent/pull/25793/commits/6c660d00f1a0c02ed7801650b8f54b994e4ef6a9